example/pythonlib/basic/1-simple/build.mill

```diff
@@ -4,9 +4,9 @@ import mill._, pythonlib._
 // Docs for PythonLib
 object foo extends PythonModule {
   object bar extends PythonModule {
-    def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+    def pythonDeps = Agg("pandas==2.2.3", "numpy==2.1.3")
   }
-  def pythonDeps = Seq("numpy==2.1.3")
+  def pythonDeps = Agg("numpy==2.1.3")
```

Comment:
This change isn't really necessary since we're planning on getting rid of `Agg` in 0.13.0 and just leaving an alias for `Seq` for backwards compatibility https://github.com/com-lihaoyi/mill/issues/3587

Reply:
Ah, didn't know that. Do you have a pointer as to what you mean with "I don't think the bespoke complexity is paying off"?